presto-docs/src/main/sphinx/security/built-in-system-access-control.rst

```diff
@@ -144,6 +145,31 @@ and deny all other access, you can use the following rules:
       ]
     }
 
+.. _query_rules:
+
+Query Rules
+-----------
+
+These rules control the ability of a user to execute, view, or kill a query. The user is
+granted or denied access, based on the first matching rule read from top to bottom. If no
+rules are specified, no checks are performed. If no rule matches, query management is not
```

Comment:
I would add to `, no checks are performed` that all access is allowed. It might be not clear what `no checks` mean.